scan_rr.go

```diff
@@ -1430,7 +1430,7 @@ func (rr *GPOS) parse(c *zlexer, o string) *ParseError {
 	return slurpRemainder(c)
 }
 
-func (rr *DS) parseDS(c *zlexer, o, typ string) *ParseError {
+func (rr *DS) parseDS(c *zlexer, _, typ string) *ParseError {
```

Comment:
a lot of functions here don't use the `o string` parameters, I left them all in that make the funcs mostly look the same.

I'll cherry pick the server.go change, that looks like a good one